src/app/views/query-runner/request/permissions/TabList.tsx

```diff
@@ -0,0 +1,25 @@
+import { DetailsList, DetailsListLayoutMode, Label, SelectionMode } from 'office-ui-fabric-react';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+const TabList = ({ permissions, columns, classes, renderItemColumn }: any) => {
```

Comment:
this is easier to write but it's taking effort to understand what the data is supposed to look like because of the lack of typing.